.github/workflows/.travis.yml

```
@@ -0,0 +1,42 @@
+name: ".travis.yml"
+
+on:
+    push:
+        branches: [ master ]
+    pull_request:
+        branches: [ master ]
+
+jobs:
+    build:
+        runs-on: ${{ matrix.os }}
+        env: 
+            TOXENV: ${{ matrix.test }}
+            RUN_INTEGRATION_TESTS: ${{ matrix.test_number}}
+        steps:
+            - uses: actions/checkout@v3
+            - run: pip install "tox"
+            - run: pip install bandit
+            - run: pip install codecov
+            - run: pip install slugs
+            - run: sudo chmod -R 777 /usr/local/lib/
+            - run: python3 setup.py install
+            - run: RUN_INTEGRATION_TESTS=${{ matrix.test_number }} ./.travis/run.sh
```

The variable here seems redundant with the variable definition above.

```suggestion
            - run: ./.travis/run.sh
```